Python/Test Data Generator/README.md

```diff
@@ -0,0 +1,47 @@
+# README
+
+## Library used for the generation of fake data
+
+### Faker
```

Comment:
Don't use a heading rather make this part of an unordered list along with its description